Makefile

```
@@ -402,6 +403,14 @@ dist-clean: clean
 	@go clean -testcache
 endif
 
+
+PHONY += functional
+functional:
```

```suggestion
functional-check:
```
Let's add check to the name

---

Done